modules/mongodb/connector.go

```diff
@@ -43,7 +43,7 @@ type mongoCollector struct {
 // `serverStatus` command.
 func (m *mongoCollector) serverStatus() (*serverStatus, error) {
 	var status *serverStatus
-	command := bson.D{{Key: "serverStatus", Value: 1}, {Key: "metrics", Value: 0}, {Key: "repl", Value: 0}}
+	command := bson.D{{Key: "serverStatus", Value: 1}, {Key: "metrics", Value: 0}, {Key: "repl", Value: 1}}
```

Comment:
@georgeok do you remember why we set it to 0? I remember we discussed something like if it is 1 it returns **much** more metrics? Or it was another request?